operators/src/processing/raster_vector_join/aggregated.rs

```diff
@@ -99,28 +107,21 @@ impl RasterPointAggregateJoinProcessor {
                 }
                 time_end = Some(raster.time.end());
 
-                let geo_transform = raster.tile_information().tile_geo_transform();
-
                 for feature_index in time_span.feature_index_start..=time_span.feature_index_end {
                     // TODO: don't do random access but use a single iterator
-                    let geometry = points.geometry_at(feature_index).expect("must exist");
-
-                    // TODO: aggregate multiple extracted values for one multi point before inserting it to the aggregator
                     let mut satisfied = false;
-                    for coordinate in geometry.points() {
-                        let grid_idx = geo_transform.coordinate_to_grid_idx_2d(*coordinate);
-
+                    for grid_idx in covered_pixels.covered_pixels(feature_index, &raster) {
                         // try to get the pixel if the coordinate is within the current tile
                         if let Ok(pixel) = raster.get_at_grid_index(grid_idx) {
```

Comment:
i think we need an else case. Only if the error is an out of bounds error we can go on, right?

Reply:
in the projection operator we do:

            Err(e) => match e {
                // Ignore errors where a coordinate is not inside a source tile. This is by design.
                GridIndexOutOfBounds {
                    index: _,
                    min_index: _,
                    max_index: _,
                }
                | InvalidGridIndex {
                    grid_index: _,
                    description: _,
                } => {}
                _ => return Err(error::Error::DataType { source: e }),
            },

Reply:
the `grid_idx` can only be `Err` if the `covered_pixels` implementation is faulty. So we could even do `expect` here, because if the implementation is faulty, we don't need the results. But I would be ok to throw some MustNotHappenException here that keeps Geo Engine running.